### PR TITLE
Angi architecture frontend

### PR DESCRIPTION
**Pull request by arbulu89**
# Description

Add architecture type visualization in the frontend, between `classic` and `angi`. Only applicable for HANA clusters.
Basically, a simple icon.
- info icon if classic
- green star icon if angi

The icon is displayed in cluster details view, checks selection view and checks execution results view.
Besides that, the cluster checks selection filters checks with the current architecture (if the cluster is angi, only bring checks with this arch type, filtering out classic specific checks). Even though we don't have such a distinction by now...

angi-icon

## How was this tested?
UT and storybook changes added

### CODE REVIEW

**Comment by angelabriel** [comment]
I like the implementation. Green Star for Angi :smiley:

File: assets/js/pages/ClusterDetails/HanaClusterDetails.test.jsx [hunk 1]
```diff
@@ -466,6 +466,53 @@ describe('HanaClusterDetails component', () => {
     }
   );
 
+  it.each([
+    { arch: 'angi', tooltip: 'Angi architecture' },
+    { arch: 'classic', tooltip: 'Classic architecture' },
+  ])(
+    'should show cluster type with $arch architecture',
```
Comment (EMaksy) [inline assets/js/pages/ClusterDetails/HanaClusterDetails.test.jsx:473]: why $ in arch ?  :)

**Review by EMaksy — APPROVED** [review]
Well done man, i guess there is only one small typo, but besides this lgtm!

File: assets/js/pages/ClusterDetails/HanaClusterDetails.test.jsx (hunk 1 above)
Comment (arbulu89) [reply assets/js/pages/ClusterDetails/HanaClusterDetails.test.jsx:473]: To interpolate the value of `arch` value coming in the scenarios

**Review by CDimonaco — APPROVED** [review]
lgtm

:chef_kiss:

**Pull request merged**